packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 0.5.0
+
+* Migrate to Google Billing Library 3.0
+  * Add `obfuscatedProfileId`, `purchaseToken` in [BillingClientWrapper.launchBillingFlow].
+  * Removed `developerPayload` in [BillingClientWrapper.acknowledgePurchase], [BillingClientWrapper.consumeAsync], [InAppPurchaseConnection.completePurchase], [InAppPurchaseConnection.consumePurchase].
+  * **Breaking Change**
+    * Removed `isRewarded` from [SkuDetailsWrapper].
```

Comment:
Is there any useful context to provide about why they are removed? E.g., link to discussion of removal in the SDK?

Reply:
added